quickstarts/microsoft.mobilenetwork/mobilenetwork-provision-proxy-sims/main.bicep

```diff
@@ -14,6 +14,11 @@ resource existingMobileNetwork 'Microsoft.MobileNetwork/mobileNetworks@2022-04-0
   name: existingMobileNetworkName
 }
 
+resource existingSimPolicyResources 'Microsoft.MobileNetwork/mobileNetworks/simPolicies@2022-04-01-preview' existing = [for item in simResources: {
```

Comment:
Remove this.

Reply:
Updated this to hold only one SIM Policy resource.